src/modules/dashboard/components/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import {
+  useColonyTransfersQuery,
+  Colony,
+  useTokenQuery,
+  useLoggedInUser,
+} from '~data/index';
+
+import { ActionButton } from '~core/Button';
+import Heading from '~core/Heading';
+import { MiniSpinnerLoader } from '~core/Preloaders';
+import NavLink from '~core/NavLink';
+import Numeral from '~core/Numeral';
+import Tooltip from '~core/Popover';
+
+import { ActionTypes } from '~redux/index';
+import { mergePayload } from '~utils/actions';
+import { checkIfNetworkIsAllowed } from '~utils/networks';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './ColonyUnclaimedTransfers.css';
+
+const displayName = 'dashboard.ColonyUnclaimedTransfers';
```

Comment:
Since this is nested under `ColonyHome`, I think this would make it so that the name contains it as well:

```suggestion
const displayName = 'dashboard.ColonyHome.ColonyUnclaimedTransfers';
```